nsv13/code/datums/keybinding/overmap.dm

```diff
@@ -272,10 +272,15 @@
 	if(!user.mob) return
 	var/mob/M = user.mob
 	var/obj/structure/overmap/OM = M.overmap_ship
+	var/wrole = FALSE //Weapon role, Because are you gunner or pilot? swap that weapon set. Why wrole? theres already a role
```

Comment:
I don't have the brains to psedocode this out in all the relevant places right now, but I'm wondering if it would be cleaner to save the user's OVERMAP_USER_ROLE_rolehere bitflag (from camera.dm's start_piloting) to the user, which would reduce these to
`OM.select_weapon(number, M.overmap_user_role)` or similar